A sample compile for Windows desktop fails

## 🐛 Bug Report

flutter run -d windows results in:

C:\VS\2022\Community\VC\Tools\MSVC\14.31.31103\include\experimental/coroutine(37,1): fatal error C1189: #error:  The <experimental/coroutine> and <experimental/resumable> headers are only supported with /await and implement pre-C++20 coroutine support. Use <coroutine> for standard C++20 coroutines. [G:\projects\tts_app\build\windows\x64\plugins\flutter_tts\flutter_tts_plugin.vcxproj]
Building Windows application...                                    36.2s
Error: Build process failed.

### Expected behavior

Does not appear to work on Windows, but compiles fine on macos.   
Might be some transient dependency, as I don't see coroutine directely used by flutter_tts.
Could be Visual C++ version related as well.

### Reproduction steps

Do a simple hello world like app using flutter_tts on windows.

### Configuration

**Version:** 4.2.1

**Platform:** 
- [x] :windows

--- a/windows/CMakeLists.txt
+++ b/windows/CMakeLists.txt
@@ -29,6 +29,10 @@ target_include_directories(${PLUGIN_NAME} INTERFACE
   "${CMAKE_CURRENT_SOURCE_DIR}/include")
 target_link_libraries(${PLUGIN_NAME} PRIVATE flutter flutter_wrapper_plugin)
 
+if(MSVC)
+    target_compile_options(${PLUGIN_NAME} PRIVATE "/await")
+endif()
+
 # List of absolute paths to libraries that should be bundled with the plugin
 set(flutter_tts_bundled_libraries
   ""

